eks/ng/role.go

```diff
@@ -196,6 +198,7 @@ Resources:
             - route53:ChangeResourceRecordSets
             - route53:DeleteHealthCheck
             Resource: "*"
+{{ if ne .ASGPolicy "" }}{{.ASGPolicy}}{{ end }}
```

Comment:
Can we add a simple test when `ASGPolicy` is not empty?

https://github.com/aws/aws-k8s-tester/blob/3e23f03490ef60feceec33f37e78b1996f39f6b9/eks/cluster/cluster_test.go#L10

Reply:
> Can we add a simple test when `ASGPolicy` is not empty?
> 
> https://github.com/aws/aws-k8s-tester/blob/3e23f03490ef60feceec33f37e78b1996f39f6b9/eks/cluster/cluster_test.go#L10

Because we are adding ASGPolicy with value in any case, do we really care ?